apps/console/src/features/core/store/reducers/config.ts

```diff
@@ -155,6 +155,7 @@ export const commonConfigReducerInitialState: CommonConfigReducerStateInterface<
             appName: "",
             appTitle: "",
             applicationTemplateLoadingStrategy: undefined,
+            isGOTEnabledForSuperTenantOnly: true,
```

Comment:
Keep default value as `undefined`
```suggestion
            isGOTEnabledForSuperTenantOnly: undefined,
```